app/listeners/automation_rule_listener.rb

```diff
@@ -89,6 +89,6 @@ def performed_by_automation?(event)
 
   def ignore_message_created_event?(event)
     message = event.data[:message]
-    performed_by_automation?(event) || message.activity?
```

Comment:
Can we add this as a condition in automation rule rather than preventing by system?